Tasks/MobileCenterDistribute/mobilecenterdistribute.ts

```diff
@@ -406,7 +406,7 @@ async function run() {
         let app = utils.resolveSinglePath(appFilePattern);
         tl.checkPath(app, "Binary file");
 
-        let continueIfSymbolsNotFoundVariable = tl.getVariable('VSMobileCenterUpload.ContinueIfSymbolsNotFound');
+        let continueIfSymbolsNotFoundVariable = tl.getVariable('MobileCenterDistribute.ContinueIfSymbolsNotFound');
```

Comment:
That's what I needed to know.  I'll revert this along with the reference in the L0 test.